packages/dart/sshnoports/bundles/universal.ps1

```diff
@@ -0,0 +1,340 @@
+        Write-Host "Pick your default region:"
+        Write-Host "  am   : Americas"
+        Write-Host "  ap   : Asia Pacific"
+        Write-Host "  eu   : Europe"
+        Write-Host "  @___ : Specify a custom region atSign"
+        while (-not ($host_atsign -match "@.*")) {
+            switch -regex ($host_atsign.ToLower()) {
+                "^(am).*" {
+                    $host_atsign = "@rv_am"
+                    break
+                }
+                "^(eu).*" {
+                    $host_atsign = "@rv_eu"
+                    break
+                }
+                "^(ap).*" {
+                    $host_atsign = "@rv_ap"
+                    break
+                }
+                "^@" {
+                    # Do nothing for custom region
+                    break
+                }
+                default {
+                    $host_atsign = Read-Host "Region"
+                }
+            }
+        }
+        $script:HOST_ATSIGN = $host_atsign
+    }
+    $clientPath = "$script:INSTALL_PATH\sshnp\$script:DEVICE_NAME$script:DEVICE_ATSIGN.ps1"
+    "sshnp.exe -f '$script:CLIENT_ATSIGN' -t '$script:DEVICE_ATSIGN' -d '$script:DEVICE_NAME' -r '$script:HOST_ATSIGN' -s -u '$Env:UserName'"  | Out-File -FilePath  $clientPath
+    if (-not (Test-Path $clientPath -PathType Leaf)) {
+        Write-Host "Failed to create client script'. Please check your permissions and try again."
+        Cleanup
+        Exit 1
+    } 
+    Write-Host "Created client script for $script:DEVICE_NAME$script:DEVICE_ATSIGN"
+}
+
+function Install-Device {
+    Write-Host "Installed at_activate and sshnpd binaries to $script:INSTALL_PATH"
+    Download-Winsw
+    New-Item -Path "$script:INSTALL_PATH\sshnp\" -Name "logs" -ItemType "directory" 
+    $servicePath = "$script:INSTALL_PATH\sshnp\sshnpd.exe"
+    $xmlPath = "$script:INSTALL_PATH\sshnp\sshnpd_service.xml"
+    [xml]$xmlContent = Get-Content $xmlPath
+    $xmlContent.service.arguments = "-a $script:DEVICE_ATSIGN -m $script:CLIENT_ATSIGN -d $script:DEVICE_NAME -k $script:homepath/.atsign/keys/$script:DEVICE_ATSIGN"+ "_key.atKeys -s"
+    $xmlEnv = $xmlContent.service.env | Where-Object {$_.Name -eq "USERPROFILE"}
+    $xmlEnv.Value = $script:homepath 
+    $xmlContent.Save($xmlPath)
+    if (-not (Test-Path $servicePath -PathType Leaf)) {
+        Write-Host "Failed to create service script'. Please check your permissions and try again."
+        Cleanup
+        Exit 1
+    } 
+    try {
+        Invoke-Expression "$script:service_path install"
+        Invoke-Expression "$script:service_path start"
+    }
+    catch {
+        Invoke-Expression "$script:service_path stop" -ErrorAction SilentlyContinue
+        Invoke-Expression "$script:service_path uninstall" -ErrorAction SilentlyContinue
+    }
+}
+
+function Uninstall-Both{
+    if (Get-Service "sshnpd" -ErrorAction SilentlyContinue){
+        Invoke-Expression "\$script:service_path stop" -ErrorAction SilentlyContinue
```

Comment:
Hm, backslash? Could be interesting problem :) 
```suggestion
        Invoke-Expression "$script:service_path stop" -ErrorAction SilentlyContinue
```

Reply:
resolved in [b354179](https://github.com/atsign-foundation/noports/pull/916/commits/b354179539a1b787943860f126b6393a4d1b899e)